### PR TITLE
Address FAIR review comments

### LINKED ISSUE #8: set_pid is not listed in the Run reference page
i.e. https://docs.simvue.io/reference/run/

### LINKED ISSUE #9: Set favicon to Simvue logo
Set favicon for docs website to be the Simvue logo. Will likely need a black version of the logo, or a white logo which has a black or navy background.

Simple change once we have the logo image, see https://squidfunk.github.io/mkdocs-material/setup/changing-the-logo-and-icons/#favicon

### LINKED ISSUE #10: Methods missing from reference docs
Some methods are missing from reference docs:

**For Run():**
`set_pid()`
`config()`

**For Client():**
`get_metrics_names()`

### LINKED ISSUE #12: get_folder() or get_folders() are not mentioned in Analysis docs
The Client class functions `get_folder()` and `get_folders()` are not mentioned on the Analysis (or Tutorial) documentation. These methods should be mentioned in those locations so that the user knows that they exist and how to use them.

### LINKED ISSUE #15: Rename 'python-client' page
'python-client' page of UI has a misleading URL, since the 'Tracking & Monitoring', 'Analysis' and 'Reference' pages all discuss the Python client.

Should change the URL to 'tracking_and_monitoring' or something similar

### LINKED ISSUE #16: Produce a vocabulary 'cheat sheet' for users to understand what each term means
Should create some kind of cheat sheet in the docs which outlines what each term used by Simvue represents. Terms such as Runs, Metrics, Alerts, Events etc should be defined. This should mostly be aimed at new users, or users which do not want to look through the whole docs just to view some runs.

### LINKED ISSUE #17: Increase links between pages
Add extra links between different pages of the UI to increase ease of use.

For example, in each section of the 'Tracking & Monitoring' page, when a new command is introduced there should be a link to the Reference documentation for it, and to the section in the Tutorial where it is used.

### LINKED ISSUE #19: Increase descriptiveness of links
Links should be descriptive on where they will take you, instead of using things like 'find more information **_here_**'. Need to go through the documentation and find any examples of these types of links, and replace them with more descriptive examples. If you read only the link text and not the surrounding descriptive text, you should still be able to tell where that link will send you.

### LINKED ISSUE #20: Add descriptions to all images
All images on the docs website need alternative descriptions, and/or to be thoroughly described by surrounding body text. The diagrams on the Welcome page need to have the surrounding text improved so that they are described in more detail, and the graphs which are produced by Simvue which are shown on other pages need descriptions adding.

### LINKED ISSUE #21: Links should be underlined as well as different colours
All links used on the docs website should be underlined as well as coloured, so that colourblind people can still recognise them

### PR DESCRIPTION
**Pull request by wk9874**
Addressing FAIR review comments:

- Added missing methods to Reference docs, closes #8 and closes #10 
- Renamed python-client, closes #15 
- Added underlining to links, closes #21 
- Increased descriptiveness of links so that they can be understood without reading any body test, closes #19
- Added terminology cheat sheet, closes #16 
- Added favicon, closes #9 
- Added alt text to images, closes #20 
- Added info about get_folder(), get_folders(), and delete_folder() to Analysis docs, closes #12 
- Added boxes for links to reference and tutorial docs to tracking & monitoring and analysis pages, closes #17

### CODE REVIEW

File: docs/analysis/metrics.md
Comment (AbyAbraham21) [inline docs/analysis/metrics.md:1]: Line 127 onwards does not look right. Please check

File: docs/analysis/metadata.md
Comment (AbyAbraham21) [inline docs/analysis/metadata.md:1]: Line 127 onwards does not look right. Please check

**Review by AbyAbraham21 — COMMENTED** [review]
further-docs "Further Documentation" seems to be missing or not referenced correctly, please can you check.

**Comment by alahiff** [comment]
I don't think having the repeated `View reference documentation` etc is necessary. For example instead of:

image

have:

image

which to me looks much better.

**Review by AbyAbraham21 — CHANGES_REQUESTED** [review]
A few changes suggested. Please update and merge

**Review by AbyAbraham21 — APPROVED** [review]

**Comment by AbyAbraham21** [comment]
Updated documentation based on FAIR review

**Pull request merged**